editor/plugins/script_editor_plugin.cpp

```diff
@@ -49,6 +49,7 @@
 #include "editor/find_in_files.h"
 #include "editor/node_dock.h"
 #include "editor/plugins/shader_editor_plugin.h"
+#include "modules/gdscript/gdscript.h"
```

Comment:
This file should stay generic and can't depend on a specific module. This would make it impossible to build Godot with `module_gdscript_enabled=no`.

If GDScript specific behavior is needed here, it needs to go through the Script/ScriptLanguage abstraction and be implemented in the GDScript module.

Reply:
OK, thanks for pointing it out.